lib/ui.js

```diff
@@ -591,16 +545,19 @@ ui.redraw.done = function () {
  * total mess. So forgive me.
  */
 
-global.__vorpal = global.__vorpal || {};
-global.__vorpal.ui = global.__vorpal.ui || {
-  exists: false,
-  exports: undefined
-};
-
-if (!global.__vorpal.ui.exists) {
-  global.__vorpal.ui.exists = true;
-  global.__vorpal.ui.exports = ui;
-  module.exports = exports = global.__vorpal.ui.exports;
-} else {
-  module.exports = global.__vorpal.ui.exports;
-}
+// TODO investigate
```

Comment:
I think this is needed for the `ui.attach` methods. I needed some global way of knowing how many instances of Vorpal I was using and ensuring I only had one UI running. This is because in multiple instances of Vorpal, initiating multiple UIs breaks everything.